src/screens/MapScreen.js

```diff
@@ -148,13 +187,46 @@ class MapScreen extends React.Component<Props, State> {
     }
     store.set('vocabById')(vocabById);
     store.set('vocabFromKeyword')(vocabFromKeyword);
-    console.log(venues);
+    // console.log(venues);
   }
 
   _onRegionChangeComplete = region => {
     this.setState({ region });
   };
 
+  // _getLocationAsync = async () => {
+  //   let { status } = await Permissions.askAsync(Permissions.LOCATION);
```

Comment:
could we add asking for permission back in? my phone is rejecting location calls by default and there's currently no indication besides a (silent) error.